### PR TITLE
Added Twake node docs

### PR DESCRIPTION
**Pull request by harshil1712**
- [x] Resize workflow.png

### CODE REVIEW

File: docs/nodes/credentials/Twake/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+2. Click on 'Main' in the top left corner.
+3. Select 'Workspace settings' from the dropdown list.
+4. Select 'Applications and connectors'
+5. Click on 'Search applications...'.
+6. Search for 'n8n' and click on 'Display'.
+7. Click on 'Install' and select 'Confirm'.
+8. Click on 'Configure' and copy the Workspace key.
+9. Use the Workspace key with your Twake node credentials in n8n.
+
+![Getting Twake workspace key](./using-access-token.gif)
```
Comment (tanay1337) [inline docs/nodes/credentials/Twake/README.md:21]: Incorrect location of the image leads to the page not being rendered.

File: docs/nodes/nodes-library/nodes/Twake/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example Usage
+
+This workflow allows you to send a message to a channel on Twake. You can also find the [workflow](https://n8n.io/workflows/595) on the website. This example usage workflow would use the following two nodes.
+- [Start](../../core-nodes/Start/README.md)
+- [Twake]()
+
+The final workflow should look like the following image.
+
+![A workflow with the Twake node](./workflow.png)
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Twake/README.md:26]: The image should be zoomed in only once and the nodes should fit within the grid (as much as they can).

File: docs/nodes/nodes-library/nodes/Twake/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+
+### 2. Twake node
+
+1. First of all, you'll have to enter credentials for the Twake node. You can find out how to do that [here](../../../credentials/Twake/README.md).
+2. Select the channel from the ***Channel ID*** dropdown list.
+3. Enter the content in the ***Content*** field.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Twake/README.md:37]: 'content of the message'

File: docs/nodes/credentials/Twake/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+## Using Access Token
+
+1. Access your [Twake](https://web.twake.app) workspace.
+2. Click on 'Main' in the top left corner.
+3. Select 'Workspace settings' from the dropdown list.
+4. Select 'Applications and connectors'
+5. Click on 'Search applications...'.
+6. Search for 'n8n' and click on 'Display'.
+7. Click on 'Install' and select 'Confirm'.
+8. Click on 'Configure' and copy the Workspace key.
```
Comment (tanay1337) [inline docs/nodes/credentials/Twake/README.md:18]: 'Workspace key' -> Is that the correct capitalization?

File: docs/nodes/nodes-library/nodes/Twake/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+4. Click on ***Execute Node*** to run the node.
+
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Twake/README.md:39]: The end of the markdown files should have just one empty newline.

File: docs/nodes/credentials/Twake/README.md (hunk 4 above)
Comment (harshil1712) [reply docs/nodes/credentials/Twake/README.md:18]: That is how they have capitalized in the Twake app.

Comment (tanay1337) [reply docs/nodes/credentials/Twake/README.md:18]: Gotcha!

**Pull request merged**